sdk/ml/azure-ai-ml/azure/ai/ml/entities/_endpoint/online_endpoint.py

```diff
@@ -363,7 +363,9 @@ class ManagedOnlineEndpoint(OnlineEndpoint):
     :param identity: defaults to SystemAssigned
     :type identity: IdentityConfiguration, optional
     :param kind: Kind of the resource, we have two kinds: K8s and Managed online endpoints, defaults to None.
-    :type kind: str, optional
+    :type kind: str, optional,
+    :param public_network_access: Whether to allow public endpoint connectivity
```

Comment:
Should we put little bit more documentation on what are the allowed values? Because this is more like Boolean with only two possible values.

Reply:
@nancy-mejia: You can look at the documentation here: https://learn.microsoft.com/en-us/azure/machine-learning/how-to-secure-online-endpoint?tabs=model